engine/src/flutter/shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -306,6 +306,17 @@ @implementation FlutterViewController {
 
 @dynamic accessibilityBridge;
 
+// Returns the text input plugin associated with this view controller.
+// This method only returns non nil instance if the text input plugin has active
+// client with viewId matching this controller's view identifier.
+- (FlutterTextInputPlugin*)textInputPlugin {
```

Comment:
I'm concerned that this name might cause trouble if a maintainer does not read the doc carefully. Shall we rename it to something like `matchingTextInputPlugin`?

Reply:
That's a good point. I'll rename it to `activeTextInputPlugin`.